base/gcutils.jl

```diff
@@ -105,6 +105,17 @@ Control whether garbage collection is enabled using a boolean argument (`true` f
 """
 enable(on::Bool) = ccall(:jl_gc_enable, Int32, (Int32,), on) != 0
 
+"""
+    GC.take_heap_snapshot(io::IOStream)
+
+Write a snapshot of the heap, in the JSON format expected by the Chrome
+Devtools Heap Snapshot viewer (.heapsnapshot extension), to the given
+IO stream.
```

Comment:
Perhaps worth adding a one-liner? 
```suggestion
    GC.take_heap_snapshot(io::IOStream)
    GC.take_heap_snapshot(filepath::String)

Write a snapshot of the heap, in the JSON format expected by the Chrome
Devtools Heap Snapshot viewer (.heapsnapshot extension), to the given
IO stream or file path.
```

Reply:
unfortunately when I tried to add `IOStream` there was a build error, since it's not defined yet… but it seems to work with a string variant and an untyped variant